storybook/src/components/document-card/document-card.content.ts

```diff
@@ -0,0 +1,6 @@
+import { html } from "lit-html";
+
+export const typeAuthority = html`<span class="type">Bestemmingsplan</span>
```

Comment:
@tfrijsewijk 

Is dit ook onderdeel van onze mark-up voorschriften? Gaan wij dit stylen? Met deze opzet zal dat wel moeten, want er is nu geen ruimte tussen type en authority:

<img width="1696" alt="Screenshot 2024-10-10 at 16 20 24" src="https://github.com/user-attachments/assets/f8c2bc3f-ec13-461f-984c-20c0e1ee2877">

Reply:
Och ook weer zoiets vreselijks 😭 

Tsja, het is wel onderdeel van de markup voorschriften. Wat denk je hiervan?

```html
<dso-document-card>
  <div slot="status">
    <span class="dso-document-card-status-item">Bestemmingsplan</span>
    <span class="dso-document-card-status-item">Gemeente Oldebroek</span>
  </div>
</dso-document-card>
```

```scss
::slotted([slot="status"]) {
  display: flex;
  gap: 8px;
}
```

ofzo.

Kom je hiermee weg? En gaat dat nog goed met de `typeToelichting`?

Een alternatief is `@Prop() statusItems: string[]` en aanroepen met `<dso-document-card .statusItems=${['Bestemmingsplan', 'Gemeente Oldebroek']}>`.

Reply:
Dit is evenwel zijn evenwel geen status items. Status is de onderste regel. Dit gaat over type en bevoegd gezag. Hence type. Hence authority. 🤷 

Dus ik maak er 
```
::slotted([slot="status"]) {
  display: flex;
  gap: 8px;
}
```
van.

En de markup:

```
<span class="dso-document-card-type">Bestemmingsplan</span>
<span class="dso-document-card-authority">Gemeente Harderwijk</span>
```

Evenwel ontgaat mij het nut en de noodzaak van die class met prefix dso. Die zijn dus van ons, maar we stylen er niets op.

Reply:
> Dit is evenwel zijn evenwel geen status items. Status is de onderste regel. Dit gaat over type en bevoegd gezag. Hence type. Hence authority. 🤷

Woeps, waar ik `status` typte had inderdaad `type` moeten staan.

> Evenwel ontgaat mij het nut en de noodzaak van die classes met prefix dso. Die zijn dus van ons, maar we stylen er niets op.

De classes `dso-document-card-type` en `dso-document-card-authority` impliceren dat "type" en "authority" entiteiten zijn waar Card wat van weet, echter hebben we alleen "type" gemodelleerd.

Mijn voorstel was:

```html
<dso-document-card>
  <div slot="type">
    <span class="dso-document-card-type-item">Bestemmingsplan</span>
    <span class="dso-document-card-type-item">Bestemmingsplan</span>
  </div>
</dso-document-card>
```

* De functionaliteit/ontwerp/modellering werkt nog steeds met het concept "type", echter kunnen het er meerdere zijn.
* Vanuit de component stylesheet kan met de `::slotted()` selector de items achter elkaar met de juiste gap worden gezet.
* Je hebt gelijk dat de classes op de spans niet nodig zijn. Dit is 'n beetje vooruitkijken is regeren. Wellicht dat we toch iets met die items willen doen.

De prefix `dso` is omdat het onderdeel van de markup voorschriften zijn, deze elementen leven in het DOM van de afnemer. En dus moeten we namespacen.

Reply:
Ok. Ik heb er dus `dso-document-card-type-item` van gemaakt.

Reply:
Fantastisch